test/components/plots/xyPlotTests.ts

```diff
@@ -27,6 +27,23 @@ describe("Plots", () => {
           .renderTo(svg);
     });
 
+    it("autorange getters", () => {
+      plot.autorange("x");
```

Comment:
Maybe test the default as well (I believe it should be "none"?). i.e. assert `plot.autorange()` before setting it explicitly